apps/api/src/app/shared/dtos/step-filter.ts

```diff
@@ -102,7 +109,8 @@ type FilterParts =
   | WebhookFilterPart
   | RealtimeOnlineFilterPart
   | OnlineInLastFilterPart
-  | PreviousStepFilterPart;
+  | PreviousStepFilterPart
+  | ITenantFilterPart;
```

Comment:
All the others are a class with the api properties, and are not imported from shared. Maybe we should stay consistent and add a TenantFilterPart here as well

Reply:
This will be solved in my next PR: https://github.com/novuhq/novu/pull/4048